src/status_im/ui/screens/navigation.cljs

```diff
@@ -59,8 +59,10 @@
 (register-handler-db
   :navigate-to
   (re-frame/enrich preload-data!)
-  (fn [db [_ new-view-id]]
-    (navigate-to db new-view-id)))
+  (fn [db [_ view-id params]]
+    (-> db
+        (assoc-in [:screen-params view-id] params)
+        (navigate-to view-id))))
```

Comment:
I would move the `screen-params` association into the `navigate-to` function (maybe overload it to optionally take screen params as last argument. 
The reason why is that in some (rare) cases, we would like to compose `navigate-to` function (capturing view + params change in app-db) with some other effect in one atomic handler, like this (example):
```clj
(register-handler-fx
  :navigate-to-wallet
  (fn [{:keys [db]} params]
    {:db                   (navigation/navigate-to :wallet params)
     :fetch-token-balances (:tokens params)}))
```
With params association outside `navigate-to`, it would force us to make 2 dispatches for one atomic action (navigating to wallet)